packages/gitlab/dev/mock-gitlab/api-v4-v15.7.0.ts

```diff
@@ -1,5 +1,55 @@
+const graphqlProjectCoverageQuery = `
```

Comment:
Please pay attention to the spaces the hit does not seem to work.

Reply:
Changed with regexp parsing and query matching